extensions/data-workspace/package.vscode.json

```diff
@@ -0,0 +1,6 @@
+{
+	"name": "data-workspace-vscode",
+	"extensionDependencies": [
+		"ms-mssql.mssql"
+	]
+}
```

Comment:
so the idea is to merge the content of this file with package.vscode?

Reply:
It will be merged with the package.json (basically the values here will overwrite those) when generating the VS Code VSIX

Reply:
we should document this somewhere. for new extensions ...

Reply:
Yeah, I'll write up a quick doc somewhere in our wiki

Reply:
https://github.com/microsoft/azuredatastudio/wiki/Developer-Getting-Started-Guide